From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zetarancio <zetarancio@users.noreply.github.com>
Date: Mon, 17 Mar 2026 00:00:00 +0100
Subject: [PATCH] net: rtl8733bu: enable LPS_LCLK deep clock gating

Uncomment CONFIG_LPS_LCLK so the MAC clock gates down to 32 kHz
during 802.11 LPS periods.  This provides maximum power savings
while associated but idle, which is the primary operating mode for
embedded devices like the Miyoo Flip.

The guard is already present in the vendor driver's autoconf.h but
was left commented out by default.  All supporting code paths
(LPS_LCLK rpwm/cpwm handshake, 32k clock switching, fw lps
deep-sleep) are already compiled when CONFIG_LPS is enabled.

Signed-off-by: Zetarancio <zetarancio@users.noreply.github.com>
---
 include/autoconf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/autoconf.h
+++ b/include/autoconf.h
@@ -205,7 +205,7 @@
 
 	#define CONFIG_LPS
 	#if defined(CONFIG_LPS)
-		/* #define CONFIG_LPS_LCLK */
+		#define CONFIG_LPS_LCLK
 	#endif
 
 	#ifdef CONFIG_LPS_LCLK
